README.md

```diff
@@ -5,27 +5,52 @@
 [![build](https://github.com/microsoft/react-native-test-app/actions/workflows/build.yml/badge.svg?event=push)](https://github.com/microsoft/react-native-test-app/actions/workflows/build.yml)
 [![npm version](https://img.shields.io/npm/v/react-native-test-app)](https://www.npmjs.com/package/react-native-test-app)
 
-- [Quick Start](https://github.com/microsoft/react-native-test-app/wiki/Quick-Start)
-- [Configuring the Test App](#configuring-the-test-app)
+- [Quick Start](#quick-start-⚡)
 - [Migrate an Existing Test App](https://github.com/microsoft/react-native-test-app/wiki/Migrate-an-Existing-Test-App)
-- [Known Issues](#known-issues)
+- [Configuring the Test App](https://github.com/microsoft/react-native-test-app/wiki/Manifest-%28app.json%29)
+- [Known Issues](#known-issues-⚠️)
 
 React Native Test App (RNTA) provides test apps for all platforms as a package.
 It handles the native bits for you so you can focus on what's important: your
 product.
 
-If you want to learn how RNTA is used at Microsoft, and see a demo of how to add
-it to an existing library - you can watch the
-["Improve all the repos – exploring Microsoft’s DevExp"](https://youtu.be/DAEnPV78rQc?t=499)
-talk by [@kelset](https://github.com/kelset) and
-[@tido64](https://github.com/tido64) from React Native Europe 2021.
+## Libraries Using RNTA 📚
```

Comment:
As a potential consumer, I'd rather look at quick start first. I think we should move this further down. Maybe below "Learn More".